build/build/src/engine/context.rs

```diff
@@ -108,6 +108,7 @@ impl RunContext {
 
     /// Check that required programs are present (if not, installs them, if supported). Set
     /// environment variables for the build to follow.
+    #[instrument(skip(self))]
```

Comment:
what does this mean?

Reply:
> what does this mean?

This introduces a [span](https://docs.rs/tracing/latest/tracing/span/index.html) coupled with the function execution.
Essentialy it logs function start and end, and groups logs from it under that span.